pyxel/editor/app.py

```diff
@@ -27,14 +27,23 @@ class App(Widget):
     def __init__(self, resource_file):
         # Get absolute path of resource file before initializing Pyxel
         resource_file = os.path.abspath(resource_file)
+        base_name = os.path.splitext(resource_file)[0]
+        palette_file = base_name + '.pyxpal'
 
         # Initialize Pyxel
         pyxel.init(APP_WIDTH, APP_HEIGHT)
         pyxel.mouse(True)
         self._set_title(resource_file)
+
         if os.path.exists(resource_file):
             pyxel.load(resource_file)
 
+        if os.path.exists(palette_file):
+            with open(palette_file) as file:
+                palette = [int(line.lstrip("#"), 16)
+                           for line in file.read().splitlines()]
```

Comment:
I could improve error reporting in case of bad file format.

Reply:
Pyxel is also designed for working as Rust module.
So palette-related things should be implemented for pyxel.load as Rust code not Python.

Reply:
But the asset loading is there in the Python code!  The editor is a Python app, not a Rust one, correct?